### PR TITLE
update aws-vpc-cni chart for v1.11.0 release

### PR DESCRIPTION
**Pull request by M00nF1sh**
### Description of changes
update aws-vpc-cni chart for v1.11.0 release

### Checklist
- [x] Added/modified documentation as required (such as the `README.md` for modified charts)
- [x] Incremented the chart `version` in `Chart.yaml` for the modified chart(s)
- [x] Manually tested. Describe what testing was done in the testing section below
- [x] Make sure the title of the PR is a good description that can go into the release notes

### Testing
```helm install aws-vpc-cni ./charts/aws-vpc-cni/ -n kube-system```

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by jayanthvn — APPROVED** [review]

File: stable/aws-vpc-cni/templates/eniconfig.yaml [hunk 1]
```diff
@@ -3,7 +3,7 @@
 apiVersion: crd.k8s.amazonaws.com/v1alpha1
 kind: ENIConfig
 metadata:
-  name: {{ required ".Values.eniConfig.region must be specified" $.Values.eniConfig.region }}{{ $key }}
```
Comment (ernetas) [inline stable/aws-vpc-cni/templates/eniconfig.yaml:6]: Why was this removed? This is a breaking change and doesn't seem to be documented anywhere?

Comment (ernetas) [reply stable/aws-vpc-cni/templates/eniconfig.yaml:6]: I opened https://github.com/aws/eks-charts/pull/739 with the intent to propose reverting this, as it seems to be an accidental change.
